contracts/src/finance/vesting/vesting_events.rs

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+/// Emitted when vested tokens are released
+#[ink::event]
+#[derive(Debug)]
+pub struct TokenReleased {
```

Comment:
add ink(topic) named "owner" / "from" : AccountId that will represent from whose vest the tokens were released.

For example, if Alice releases a token directly to Bob then:
owner/from: Alice
to: Bob